lib/generateMarkdown.js

```diff
@@ -74,20 +72,31 @@ function generateMarkdown(options) {
 
         // Render section for each property
         md += createPropertiesDetails(schema, title, options.headerLevel + 1, options.suppressWarnings);
+        md += createPropertiesDetails(schema, title, options.headerLevel + 1, options.suppressWarnings, 'definitions');
     }
 
     return md;
 }
 
 ////////////////////////////////////////////////////////////////////////////////
 
-function createPropertiesSummary(schema, suppressWarnings) {
+function createPropertiesSummary(schema, suppressWarnings, ptype) {
     var md = '';
-    md += style.propertiesSummary('Properties') + '\n\n';
+    var title;
+    var properties;
+    if (ptype === 'pattern') {
+        if (!schema.hasOwnProperty('patternProperties')) return '';
```

Comment:
Even for one-liner if statements, please wrap the body within brackets for long-term code-maintenance